### ISSUE TITLE
Smoke test for Stanford Sentiment data set

### ISSUE THREAD

**Issue by wpm** [wpm/mycroft#20]
Before trying to reproduce published results #11 just get a set of results for the Stanford Sentiment data set. This is a smoke test to give me some confidence that this is working for release 1.0.0.

Do this an experimental grid for a few different model/hyperparameter combinations on the coarse (either "positive" or "negative") sentiment samples. Do an 80/20 split and measure validation accuracy and loss.

These were run on a G1 GPU server on [Floydhub](https://www.floydhub.com) with 4 cores, 1 GPU, 61G memory, and 12G GPU memory.

| Model  | Parameters | Time | Accuracy | Loss | URL |
| ------- | ----------- | ---------- | ---- | --- | ----- |
| svm      |  defaults     |     1:08:28      |    **0.81020**     |    1.14729     |  [mycroft/9](https://www.floydhub.com/billmcn/projects/mycroft/9)     |
| nbow    | batch=32   |    2:38       |    0.70025     |   1.91377      |   [mycroft/10](https://www.floydhub.com/billmcn/projects/mycroft/10)    |
| nbow    | batch=64   |    1:33        |   0.70969     |   1.96907      |   [mycroft/11](https://www.floydhub.com/billmcn/projects/mycroft/11)    |
| nseq    | batch=32, units=32 |   2:26:52      |     0.80131      |   2.45249      |    [mycroft/5](https://www.floydhub.com/billmcn/projects/mycroft/5)   |
| nseq    | batch=32, units=64 |   2:28:39      |     0.80594     |   2.40330      |    [mycroft/6](https://www.floydhub.com/billmcn/projects/mycroft/6)   |
| nseq    | batch=32, units=256 |   2:59:28    |     0.80365   |    2.37600     |    [mycroft/7](https://www.floydhub.com/billmcn/projects/mycroft/7)   |
| nseq    | batch=256, units=64 |   24:18     |   0.75026      |    2.53510     |    [mycroft/13](https://www.floydhub.com/billmcn/projects/mycroft/13)   |

Conclusions:
* SVM and sequential models are about the same. Both outperform a neural bag of words model.
* SVM trains a lot faster.
* The literature says 80% is a decent baseline for this data set, so the code does appear to be working.

**Comment by wpm** [comment]
I'm convinced the code doesn't have any obvious bugs in it. It gets close to known baselines on the Stanford data set.

**Comment by wpm** [comment]
One more test with a GRU

| Model | Parameters | Time | Accuracy | Loss | URL |
| ------ | ----------- | ----- | ---------- | ----- | ---  |
| nseq   | GRU, batch=32, units=64 | 2:03:13 | 0.80273 |  2.52183 | [mycroft/14](https://www.floydhub.com/billmcn/projects/mycroft/14) |

Basically the same as the LSTM. The training time is a little faster.
